### PR TITLE
Add z.describe support

### PR DESCRIPTION
**Pull request by dosco**

### CODE REVIEW

File: lib/src/getZodiosEndpointDefinitionList.ts [hunk 1]
```diff
@@ -245,6 +245,9 @@ export const getZodiosEndpointDefinitionList = (doc: OpenAPIObject, options?: Te
                         paramSchema = paramItem.schema;
                     }
 
+                    const description = paramItem.description ?? "";
+                    paramSchema = { ...paramSchema, description };
```
Comment (astahmer) [inline lib/src/getZodiosEndpointDefinitionList.ts:249]: just set the paramSchema property, no need to spread there I think

**Review by astahmer — CHANGES_REQUESTED** [review]

**Comment by astahmer** [comment]
can you add a specific test for this & add a [changeset](https://github.com/changesets/changesets/blob/main/docs/intro-to-using-changesets.md) so I can merge it ?

**Comment by dosco** [comment]
I added a test but now I see that over 20 tests fail cause now the descriptions cause the zod to have a `describe()` and the tests are purely string matching on large text blobs. Do you have a suggestion short of me having to go update all these tests?

**Comment by dosco** [comment]
Also I've added a change set (not familiar with this) using `npx changeset add` and the test.

**Comment by astahmer** [comment]
> 

you can try to update the tests using vitest (I think you just need to hit the "U" key) and check that everything looks correct, then you should be good to go !

**Comment by dosco** [comment]
> > 
> 
> you can try to update the tests using vitest (I think you just need to hit the "U" key) and check that everything looks correct, then you should be good to go !

Thanks, I was new to most of tooling in this repo, like vitest, pnpm, preconstruct, etc so good to learn some new stuff. I tried everything however I can't figure out why `samples.test.ts` and `callback-example.yaml` is failing.  All other tests are passing.

**Comment by astahmer** [comment]
Screenshot 2023-05-30 at 21 44 05

looking at the CI, there seems to be an issue with the pnpm-lock file, could you try deleting it & running `pnpm i` afterwards ? it should fix it and then i'll be able to see the failing tests

**Comment by dosco** [comment]
I tried that I don't get any "fixed" file I just get the same one back so nothing to commit. I ended up manually reverting the file to the version in your repo (Hope that works). Whatever breaking change this was it was caused by a simple `pnpm i` I recommend  just using `npm` which is way more used and tested.

https://github.com/dosco/openapi-zod-client/commit/a511340c518cb20d8c4baa7e22847c72314f097e#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bb

**Comment by astahmer** [comment]
I made the appropriate changes here, i'll close this one and merge the new one
https://github.com/astahmer/openapi-zod-client/pull/148

**Pull request closed without merging**